### PR TITLE
Edits Arrival Cafe

### PR DESCRIPTION
**Pull request by orelbon**
Makes the Arrivals checkpoint viable.
capture

🆑 
tweak: Edits arrival cafe to make security checkpoint viable
/🆑

### CODE REVIEW

**Comment by Markolie** [comment]
I'd recommend moving the vent/scrubber into the carpeted area and adding a new vent/scrubber at the corner leading to botany. Right now it'll be very hard to repressurize arrivals.

**Comment by orelbon** [comment]
Good idea, ill get that fixed.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -3636,10 +3636,10 @@
...
 "brW" = (/obj/machinery/alarm{dir = 1; icon_state = "alarm0"; pixel_y = -22},/obj/item/weapon/twohanded/required/kirbyplants{tag = "icon-plant-22"; icon_state = "plant-22"},/obj/machinery/light,/turf/simulated/floor/wood,/area/hallway/secondary/entry)
 "brX" = (/obj/machinery/firealarm{dir = 2; pixel_y = -24},/obj/structure/table/woodentable,/obj/item/weapon/reagent_containers/food/drinks/coffee,/turf/simulated/floor/wood,/area/hallway/secondary/entry)
-"brY" = (/obj/structure/stool/bed/chair/office/dark{dir = 4},/turf/simulated/floor/wood,/area/security/vacantoffice)
-"brZ" = (/obj/structure/stool/bed/chair/office/dark{dir = 8},/turf/simulated/floor/wood,/area/security/vacantoffice)
-"bsa" = (/obj/machinery/alarm{pixel_y = 23},/turf/simulated/floor/wood,/area/security/vacantoffice)
-"bsb" = (/obj/machinery/atmospherics/unary/vent_scrubber,/turf/simulated/floor/wood,/area/security/vacantoffice)
+"brY" = (/obj/machinery/camera{c_tag = "EVA Airlock"; dir = 1},/turf/simulated/floor/plasteel,/area/hallway/secondary/entry)
```
Comment (Markolie) [inline _maps/map_files/cyberiad/cyberiad.dmm]: Don't forget to update the camera tag (make sure it's consistent with other ones in the area).

**Comment by SamHPurp** [comment]
I can't exactly see from the screenshot but it looks as though the AI won't be able to see some tiles at the very bottom middle.

**Comment by AndriiYukhymchak** [comment]
maybe swap vendors and wall, so vendors are _inside_ the "cafe"?

**Comment by orelbon** [comment]
@SamHPurp That was fixed after the screenshot was taken, check the commits.

**Comment by FlattestGuitar** [comment]
@orelbon update your screenshots, then. >:C

**Comment by orelbon** [comment]
Fiiiinnne

**Comment by Anasari** [comment]
Look nice

**Comment by Lady-Luck** [comment]
I guess I'll be the voice of dissent then... why bother? The Security checkpoint isn't used and likely never will be, even with this change. It's a bit of a fossil. All this does is introduce a chokepoint into arrivals that's going to make life harder for antags fleeing from security.

**Comment by orelbon** [comment]
Well, why not bother? That area as it is now is somewhat ugly and bland. The reason the security checkpoint might not be used now might have been because there was no point when people could just bypass it without a problem, if anything, this lets security/players do some role playing if they get bored. Finally, in regards to this contributing to a choke point, all of the halls in the station can be considered choke points, and even then i rarely see security organized enough to take advantage of it. There are maintenance access points all near that area, as well as being able to break the windows if the hallway gets blocked.

**Comment by Lady-Luck** [comment]
Er.. no, it's because there's no longer any point to the arrivals checkpoint.

**Comment by orelbon** [comment]
Ive found it useful for getting a recharger in an emergency when i don't have access to one. Also, as @SamHPurp has point out in the first PR, it can be used by security in a pinch when they need locker gear, even if it's seldom. And really, in the end, until someone has a better idea of what to replace it with, or enough people complain, the check point isn't going anywhere, so it might as well not be totally redundant.

**Comment by SamHPurp** [comment]
as @orelbon eluded to, I mentioned that it has saved my bacon more than once when I late joined as a sec officer, and Security had been compromised for a variety of reasons. I was able to gear up and actually have a chance.

The checkpoint should remain.

However I'm not entirely sure I can endorse this PR, as I like having both routes around the arrivals part... plus I like symmeticallity. Remaining on the fence.

**Comment by SamHPurp** [comment]
Also, I feel as though the arrivals 'hallway' going from 2 wide to 3 wide is a bad design. It should remain 3 wide, or higher.

**Comment by Lady-Luck** [comment]
Ya'll are proving my point, the arrivals checkpoint isn't actually an arrivals checkpoint anymore, it's just a backup storage closet for security. So why bother pretending it has a use by mapping around it?

I really don't like introducing a narrow chokepoint at arrivals for no reason.

But whatever. It's not a big deal either way. I just don't see the merit.

**Comment by Taaketa** [comment]
I have to agree with @Lady-Luck this is pretty ugly and I don't feel it's needed. Really ruins the symmetry of the station but taking out an obvious path. Either bring the spare office up in line with the port primary corridor or have the cafe dig into it.

But considering this is one of the areas of the station that is the least populated and has little reason to be used other than people passing through. I can't agree that this change is even needed. The security checkpoint isn't even related to it's old purpose anymore and I hope to god it never comes back for it's purpose as it was one of the worst defined jobs on the station and became troll central after a while.

**Comment by orelbon** [comment]
Alright i am starting to reconsider this, ill just leave it up for now and see how the reactions play out. Please do leave a reaction on the post to measure public opinion.

**Comment by Fruerlund** [comment]
Remove the lower two windows in the left side. So people have an alternative way to go. 

And the claims that just because its not used doesnt mean you should attempt to redo the area in order to draw more attention to it, is just... not a valid opinion IMO.

**Comment by orelbon** [comment]
Im gonna close this, ill just work on improving the look of the area instead of trying to re map it.

**Pull request closed without merging**